frontend/interfaces/software.ts

```diff
@@ -280,18 +299,21 @@ export interface IHostSoftware {
   app_store_app: IHostAppStoreApp | null;
   source: string;
   bundle_identifier?: string;
-  status: SoftwareInstallStatus | null;
+  status: Exclude<SoftwareInstallStatus, "uninstalled"> | null;
   installed_versions: ISoftwareInstallVersion[] | null;
 }
 
 export type IDeviceSoftware = IHostSoftware;
 
-const INSTALL_STATUS_PREDICATES: Record<SoftwareInstallStatus, string> = {
-  failed: "failed to install",
-  failed_install: "failed to install",
+const INSTALL_STATUS_PREDICATES: Record<
+  SoftwareInstallStatus | "pending",
+  string
+> = {
+  pending: "pending",
```

Comment:
Is `pending` being retained for for backwards compatibility? Does this still generate the expected text in the `getInstallStatusPredicate` function?

Reply:
I'll double check and clean this up a bit if it's not used anymore on an iterative PR, thanks for catching